examples/bond/test/bond-token-flow.js

```diff
@@ -115,15 +115,7 @@ describe('Bond token Tests', function () {
     console.log('Issuer Address: ', issuerLsigAddress);
 
     // fund escrow with some minimum balance first
-    const fundEscrowParam = {
-      type: types.TransactionType.TransferAlgo,
-      sign: types.SignType.SecretKey,
-      fromAccount: master.account,
-      toAccountAddr: issuerLsigAddress,
-      amountMicroAlgos: minBalance + 10000,
-      payFlags: {}
-    };
-    runtime.executeTx(fundEscrowParam);
+    runtime.fundAccount(master.account, issuerLsigAddress, minBalance + 10000);
```

Comment:
How abouot renaming `fundAccount` to `fundLsig`? It will be same as deployer API, and we update all instances in  examples  where we're funding an escrow contract. what do you think?

Reply:
i thought so, but fundAccount is more general, user can use it to fund accounts/lsig accounts. I don't have preference in both cases. what do you think?

Reply:
I would prefer to keep the name as `fundLsig`. In algob scripts user uses `executeTransaction.TransferAlgo` to fund other accounts, and will use `deployer.fundLsig` for escrow contracts

Reply:
ok will remove more generalised name.